appeals/api/src/server/repositories/service-user.repository.js

```diff
@@ -1,8 +1,19 @@
 import { databaseConnector } from '#utils/database-connector.js';
 
+/** @typedef {import('@pins/appeals.api').Appeals.UpdateAddressRequest} UpdateAddressRequest */
 /** @typedef {import('@pins/appeals.api').Schema.ServiceUser} ServiceUser */
 /**
-
+ * @typedef {Object} ServiceUserInput
+ * @property {string} [organisationName]
+ * @property {string} [salutation]
+ * @property {string} [firstName]
+ * @property {string} [middleName]
+ * @property {string} [lastName]
+ * @property {string} [email]
+ * @property {string} [website]
+ * @property {string} [phoneNumber]
+ * @property {number} [addressId]
```

Comment:
I'd expect there to be Prisma types that cover this?

perhaps: `import('#db-client').Prisma.ServiceUserCreateInput` or similar

Reply:
I think this was pre-existing - I have a stash where I've started to go through but little progress so far...